domains/zhp.pl.d/wielkopolska.js

```diff
@@ -15,5 +15,7 @@ D_EXTEND('zhp.pl',
     Delegation_A('gniezno', '178.32.205.96'),
     Delegation_A('poznangrunwald', '94.23.27.27'),
 
-    Delegation_NS('wzlot', ['ns1.atthost.pl.', 'ns2.atthost.pl.'])
-);
+    Delegation_NS('wzlot', ['ns1.atthost.pl.', 'ns2.atthost.pl.']),
+
+    Delegation_NS('wlkp', ['ns1.zenbox.pl', 'ns2.zenbox.pl'])
```

Comment:
```suggestion
    Delegation_NS('wlkp', ['ns1.zenbox.pl.', 'ns2.zenbox.pl.'])
```
DNS-y muszą kończyć się kropką, bo to jest pełen adres, nie subdomena. Dlatego CI się wyłożyło przy próbie zbudowania

Wystawiając PR nie nadałeś mi uprawnień do brancha źródłowego (był taki checkbox), więc poprawić musisz sam ;)